### DISCUSSION TITLE
Ignore Scrollbar When below some div

### DISCUSSION THREAD

**Discussion by blitzve0** [idiotWu/smooth-scrollbar#448] [Q&A] [answered]
**Summary**
New Day New Issue 😅
Hi, I have a Menu Button On Top of Scrollbar, It's a necessity to use **_document.body_** as container and i use offset.y listener to set fixed position for Menu. 

**Problem**
I have attached a Video Showing My Problem. I don't want the scrollbar to get focused when i am hovering on Menu Button. I have tried z-index , pointer-events nothing seems to be working

https://user-images.githubusercontent.com/67477131/154057532-f581685b-eacf-4505-ba10-b1979a52cf75.mp4

**Current Solution**
NONE

**HELP WANTED**
How to Not Focus on Scrollbar when Hovering on Menu but focus all other time
Any Solution is Appreciated

**Comment by dolphin-wood** [comment, marked as answer]
A quick&dirty fix:

```js
menu.addEventListener('pointerenter', () => {
  scrollbar.track.xAxis.element.style.pointerEvents =
  scrollbar.track.yAxis.element.style.pointerEvents = 'none';
});

menu.addEventListener('pointerleave', () => {
  scrollbar.track.xAxis.element.style.pointerEvents =
  scrollbar.track.yAxis.element.style.pointerEvents = '';
});
```

**Reply by blitzve0** [reply]
Everything's Fair in Love and Web Development 😂
This Worked
